<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>son</class>
 <widget class="QWidget" name="son">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>股票买卖界面</string>
  </property>
  <widget class="QPushButton" name="getbackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一个页面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(+为卖出，-为买入)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>308</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请输入你要买/卖出的份额：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signalButton">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="iconSize">
       <size>
        <width>13</width>
        <height>13</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="amountEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
